### PR TITLE
Requirement 1.2.5 Product Breakdown

### PR DESCRIPTION
**Pull request by aimeekang**
Requirement 1.2.5 Product Breakdown

### CODE REVIEW

File: client/source/components/RR/ReviewTile.jsx
Comment (AedonGrunGott) [inline client/source/components/RR/ReviewTile.jsx:107]: Not a merge blocker. Just by switching between sort options, I am able to set something as helpful multiple times.

File: client/source/components/RR/ProductBreakdown.jsx
Comment (AedonGrunGott) [inline client/source/components/RR/ProductBreakdown.jsx:33]: Is Fresh & Clean! 🧼

File: client/source/components/RR/ReviewRating.jsx
Comment (AedonGrunGott) [inline client/source/components/RR/ReviewRating.jsx:27]: Placeholder, to be replaced! Works as is though, so useful placeholder.

File: client/source/components/RR/ReviewTile.jsx
Comment (AedonGrunGott) [inline client/source/components/RR/ReviewTile.jsx:55]: Not a merge blocker. There is a _lot_ going on in this render section. I'd suggest breaking it up into smaller sub-components.

**Review by AedonGrunGott — APPROVED** [review]
Approved-stamp-illustration-on-transparent-background-PNG

File: client/source/components/RR/ReviewTile.jsx
Comment (aimeekang) [reply client/source/components/RR/ReviewTile.jsx:107]: Hm, that is a good point - I have sortType set as a state, so every time it changes, it re-renders the list, as I'm using the sort parameter of the API. I'll probably have to look in how to cache the data. Thoughts?

**Comment by aimeekang** [comment]
@BrettEastman or @avinashi10 Would you mind taking a look at the latest commits after I got August's approval? 

Main changes: I restructured the meta data that is needed to calculate average rating, added a file for helper functions (currently only contains a function to calculate the average rating), and updated the code on the Review Module so it doesn't repeat with what's happening in the App Holder. The plan is that the meta data is exposed via the context provider, and whoever needs to calculate the average rating can use the helpers.js file for it.

**Comment by BrettEastman** [comment]
I checked it out and it looks good to me. I tested the helpful button and it only incremented once, which seems good.

**Pull request merged**
